features/step_definitions/api-token.js

```diff
@@ -167,4 +170,17 @@ module.exports = function server() {
     this.Then(/^the token's 'last used' property will not be updated$/, () => {
         Assert.strictEqual(this.updatedToken.lastUsed, this.testToken.lastUsed);
     });
+
+    this.When(/^they revoke the token$/, () =>
+        request({
+            uri: `${this.instance}/${this.namespace}/tokens/${this.testToken.id}`,
+            method: 'DELETE',
+            auth: {
+                bearer: this.jwt
+            }
+        }).then(response => Assert.strictEqual(response.statusCode, 204)));
+
+    this.Then(/^the login attempt fails$/, () => {
+        Assert.strictEqual(this.loginResponse.statusCode, 401);
```

Comment:
Ok related to the above - https://github.com/screwdriver-cd/screwdriver/blob/35249f0112292ab502e8fbcabc57ce057c18d4c2/features/step_definitions/api-token.js#L45 aren't you logging out here instead of in?